src/hooks/__test__/hook-test-utils.js

```diff
@@ -0,0 +1,12 @@
+import * as RTH from '@testing-library/react-hooks';
+import { QueryClient, QueryClientProvider } from 'react-query';
+
+const ReactQueryProvider = ({ children }) => (
+  <QueryClientProvider client={new QueryClient()}>
+    {children}
+  </QueryClientProvider>
+);
+
+export function renderReactQueryHook(hook, params) {
+  return RTH.renderHook(() => hook(params), { wrapper: ReactQueryProvider });
+}
```

Comment:
In order to try to maintain the same API of the `renderHook` function from `testing-library/react-hooks`, my suggestion is that we change the `renderReactQueryHook` a little bit:
```suggestion
import { renderHook } from '@testing-library/react-hooks';
import { QueryClient, QueryClientProvider } from 'react-query';

const ReactQueryProvider = ({ children }) => (
  <QueryClientProvider client={new QueryClient()}>
    {children}
  </QueryClientProvider>
);

export function renderReactQueryHook(hook, options = {}) {
  return renderHook(hook, { ...options, wrapper: ReactQueryProvider });
}
```

Then instead of:
```js
const renderUseCompaniesSelectionMenuItem = () =>
  renderReactQueryHook(useCompaniesSelectionMenuItem, mockedCompanies);
  ```
we would do:
```js
const renderUseCompaniesSelectionMenuItem = () =>
  renderReactQueryHook(() => useCompaniesSelectionMenuItem(mockedCompanies));
```